dist/carousel/carousel.css

```diff
@@ -118,10 +117,10 @@ div.carousel {
   z-index: 1;
 }
 .carousel__control--prev {
-  left: 0;
+  left: -16px;
```

Comment:
I think that's more to do with Storybook. Can you try in the isolated view?

Reply:
I see the same issue in the isolated view. You don't see this on your end? 
<img width="814" alt="image" src="https://github.com/eBay/skin/assets/1675667/6362fd59-3dd3-4a31-af89-9282ec8761a7">

Reply:
No, I don't see it
<img width="1144" alt="Screen Shot 2023-08-15 at 10 59 49 AM" src="https://github.com/eBay/skin/assets/38065/202bf5d6-264d-4920-9ac9-e89539857bc7">

Reply:
That is very puzzling. And you're looking at the "Hidden Scrollbar" story?

Reply:
Ahhhh, no I was not looking at Hidden Scrollbar story. Thank you. Puzzle solved :-)

@agliga I think we found the reason it was there. Can you shed more light on this hidden scrollbar story? We can do it offline maybe...

Reply:
@agliga Know anything about this rule:

https://github.com/eBay/skin/blob/1925b-imcburnie/src/less/carousel/carousel.less#L333

It's at the module root, meaning buttons will be cut off wherever they are in DOM.

Reply:
Oof this is tricky because `overflow: hidden` is on root to hide the scrollbar for hidden scrollbar version. I think short term solution for now is hidden-scrollbar version of carousel keeps the paddles inside the carousel, not overlapped. 

I think this is a module we come back to and refactor in v17.

Reply:
Yea, that's a good plan. I just created a `17.0.0` milestone. We can bucket this (and other breaking changes) in there for now.

Reply:
Done.